packages/components/flex/docs.mdx

```diff
@@ -0,0 +1,90 @@
+---
+name: Flex
+route: /components/flex
+menu: Components
+---
+
+import { Playground, Props } from "docz"
+import Flex from './index'
+import Box from "../box"
+
+# Box
+
+Box component
+
+The HTML DOM structure is
+
+```
+  <div></div>
```

Comment:
with `as` prop it can be anything. Not necessarily div. Be more specific here.

Reply:
Also about Box that can be used as span, form but not P or button tag. Instead for those use the design system components.